### DISCUSSION TITLE
ZBDongle-E  come router

### DISCUSSION THREAD

**Discussion by Subia75** [indomus/forum#2589] [Hardware & Firmware]
Ciao,
ho provato la vostra guida per trasformare  in 'router' una chiavetta ZBDongle-E, per estendere la copertura.
Il problema è che non viene poi rilevata dalla mia rete.
Immagino che sia sufficiente alimentarla perchè vada in pairing o sbaglio ?
grazie

**Comment by indomus** [comment]
> Ciao, ho provato la vostra guida per trasformare in 'router' una chiavetta ZBDongle-E, per estendere la copertura. Il problema è che non viene poi rilevata dalla mia rete. Immagino che sia sufficiente alimentarla perchè vada in pairing o sbaglio ? grazie

corretto. se non hai sbagliato firmware, appena la alimenti si mette in pairing. chiaro che anche il tuo software bridge/gateway dev'essere in pairing, per farla entrare

**Reply by Subia75** [reply]
grazie per la risposta,
sono sicuro, ho provato la procedura penso 3 o 4 volte, rimettendo il firmware originale e poi quello 'router', prima di chiedere, ma non ne vuole proprio sapere.
ho provato anche con secureCRT a caricarlo giusto per farlo in locale, ma niente.

giusto per info, il file in questione è:   Z3RouterUSBDonlge_EZNet6.10.3_V1.0.0.gbl
